index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>My Gallery</title>
+</head>
+
+<body>
+    <div class="profile">
+        <img class="profile-picture" src="img/profilePicture.jpg" alt="Profile picture">
+        <h1 class="profile-name">Ihor Marchuk</h1>
+        <p class="profile-place">Kyiv, Ukraine</p>
+
+        <div class="profile-socials">
+            <div class="profile-socials-item">
+                <h2 class="count">3</h2>
+                <p class="count-text">Posts</p>
+            </div>
+            <div class="profile-socials-item">
+                <h2 class="count">98</h2>
+                <p class="count-text">Followers</p>
+            </div>
+            <div class="profile-socials-item">
+                <h2 class="count">113</h2>
+                <p class="count-text">Following</p>
+            </div>
+        </div>
+    </div>
+
+    <div class="images">
+        <picture class="image-1">
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture

Picture element is used to define several sources for image to let browser decide what image should be loaded.

In this case we don't use any other sources, so we don't need picture element here